app/lib/efile/tax_calculator.rb

```diff
@@ -58,28 +58,28 @@ def set_line(line_id, value_fn_or_data_source, field = nil)
       @lines[line_id].value_access_tracker = @value_access_tracker
     end
 
-    def filing_status_mfj?
+    def state_filing_status_mfj?
       @filing_status == :married_filing_jointly
     end
 
-    def filing_status_mfs?
+    def state_filing_status_mfs?
       @filing_status == :married_filing_separately
     end
 
-    def filing_status_single?
+    def state_filing_status_single?
       @filing_status == :single
     end
 
-    def filing_status_hoh?
+    def state_filing_status_hoh?
       @filing_status == :head_of_household
     end
 
-    def filing_status_qw?
+    def state_filing_status_qw?
       @filing_status == :qualifying_widow
     end
 
-    def md_filing_status_dependent?
-      @direct_file_data.claimed_as_dependent?
+    def state_filing_status_dependent?
+      @filing_status == :dependent
```

Comment:
just want to nuke all these methods and use `intake.state_filing_status_{filing_status}` directly instead of relying on the calculators...